bigtable/google/cloud/bigtable/table.py

```diff
@@ -276,6 +284,31 @@ def read_rows(self, start_key=None, end_key=None, limit=None,
         # We expect an iterator of `data_messages_v2_pb2.ReadRowsResponse`
         return PartialRowsData(response_iterator)
 
+    def mutate_rows(self, rows):
+        """Mutates multiple rows in bulk.
+    
+        :type rows: list
+        :param rows: List or other iterable of :class:`.DirectRow` instances.
+        
+        :rtype: list
+        :returns: A list of tuples (``MutateRowsResponse.Entry`` protobuf 
+                  corresponding to the errors, :class:`.DirectRow`) 
```

Comment:
It tries to indicate that each element of the list is a tuple with 2 values: the first one is an instance of ``MutateRowsResponse.Entry`` and the second one is DirectRow instance itself.
Should I change description or return type in some way?